bin/node/runtime/src/lib.rs

```diff
@@ -232,6 +232,7 @@ impl pallet_randomness_collective_flip::Config for Runtime {}
 impl pallet_utility::Config for Runtime {
 	type Event = Event;
 	type Call = Call;
+	type PalletsOrigin = OriginCaller;
```

Comment:
Updated in scope of ["allowing root to dispatch on behalf other origin"](https://github.com/paritytech/substrate/pull/9934)